js/main.js

```diff
@@ -1076,6 +1246,13 @@ function loadGameData() {
         replaceSaveDict(gameData.itemData, gameDataSave.itemData);
         replaceSaveDict(gameData.battleData, gameDataSave.battleData);
 
+        for (let id in gameDataSave.currentOperations){
+            gameDataSave.currentOperations[id] = moduleOperations[id];
+        }
+        for (let id in gameDataSave.currentModules){
+            gameDataSave.currentModules[id] = modules[id];
+        }

```

Comment:
Fehlen da nicht die eigentlichen Spielwerte dann in gameDataSave?

Reply:
Ne das sind ja nur die 'current'. Das könnten auch einfach Ids sein sind aber gerade halt Referenzen

Reply:
Aber das interne leben der Module wird hier noch nicht wiederhergestellt (on/off… current mode).